MANIFEST.in

```diff
@@ -0,0 +1,9 @@
+include *.rst
+include *.txt
+include tox.ini
+include bootstrap.py
+include buildout.cfg
+
+recursive-include src *.txt
+
+global-exclude *.pyc
```

Comment:
This file is missing the final newline.

(It probably doesn't matter.  Git shows it as a whitespace error.  I don't expect distutils will care.)